### DISCUSSION TITLE
Begginer: help to undeerstand README and finish the script

### DISCUSSION THREAD

**Discussion by shaiyaofolena** [community/community#109881] [New to GitHub]
Hello everyone,
I'm finally getting into GitHub =) So far I've just done HTML CSS and a bit of very basic PHP.

My first challenge: being able to download videos hosted by Kinescope using this GitHub script https://github.com/Prividen/kinescope-downloader?tab=readme-ov-file#kinescope-downloader

I installed scoop and ffmpeeg using this tutorial: https://amiaopensource.github.io/ffmpeg-artschool/#:~:text=Check%20if%20you%20have%20ffmpeg,will%20have%20to%20install% 20ffmpeg.

But now I'm lost! what should I do to continue ? The READMEE file from https://github.com/Prividen/kinescope-downloader?tab=readme-ov-file#kinescope-downloader is not detailed enough for a beginner like me :( I don't understand the next step I have to do.

Can someone help me understand the next steps or find a link to a tutorial that can help me finish the script?

Thank you all in advance

**Comment by VISAGRIBANKd5** [comment]
![_](https://github.com/community/community/assets/158062303/e017bb4b-1b87-4a54-b6d9-35adeb470eec)
hình ảnh trong Videos ngắn tích hợp phòng trong thủy NGUYỄN HẢI Đăng Birthday 28/05 /1984 IDVMNO840045432079084004543/QR/chio079084004543/XRAY/_-SCANNER _TLD/VND. Thb

**Comment by VISAGRIBANKd5** [comment]
Hình ảnh trong ứng dụng pintersent cá nhân App Market Android 11 OPPOA16K CPH2349 có thể là bản quyền cá nhân khác của tôi

**Comment by VISAGRIBANKd5** [comment]
Black Newrom Intel® Bitcoin®
